### DISCUSSION TITLE
导出数据文档和实际操作不一致

### DISCUSSION THREAD

**Discussion by luanshidujun** [apache/iotdb#4284] [Q&A]
文档中导出数据有-q指令，但实际操作报错，版本V0.12.2
![image](https://user-images.githubusercontent.com/37785928/139616911-eaf1f70a-3ab0-4a62-a277-f8b1f985feff.png)
![image](https://user-images.githubusercontent.com/37785928/139616932-b5613ca4-7f04-468f-956c-b9c971748b0e.png)

**Comment by HTHou** [comment]
您好，这个是因为csv的 -q 功能是即将发布的0.12.3版本里新增的，0.12.2是不支持的。
